src/Logger.js

```diff
@@ -42,7 +42,7 @@ class Logger {
       return;
     }
 
-    this.write(this.chalk.yellow(message));
+    this.status(emoji.warning, message, 'yellow');
```

Comment:
This introduces a logging defect. If I'm using a `logLevel` of 2, `warn` should write. With this change, it will be silenced by the check in `this.status` which requires a `logLevel` of 3. 

A test for such a regression is included in #562.